sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -1044,7 +1044,12 @@ def _add_argparse_args(cls, parser):
             'form {"os": "<OS>", "arch": "<ARCHITECTURE>", "command": '
             '"<process to execute>", "env":{"<Environment variables 1>": '
             '"<ENV_VAL>"} }. All fields in the json are optional except '
-            'command.'))
+            'command.\n\nPlease consider using the following '
+            'options instead, depending on the environment type:\n'
+            ' --docker_env_container_image\n'
+            ' --process_env_command\n'
+            ' --process_env_variables\n'
+            ' --external_env_service_address'))
```

Comment:
Could we change the option name pattern to `env_<type>_<option>`?

Reply:
Done.